pkgs/by-name/qu/quiet/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchurl
+, appimageTools
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "quiet";
+  version = "1.9.0";
+
+  src = fetchurl {
+    url = "https://github.com/TryQuiet/quiet/releases/download/quiet@${version}/Quiet-${version}.AppImage";
+    hash = "sha256-2rrm8ejuDR1yjj5qpmKLYMniuirTQEtMTd0pUGkgEhU=";
+  };
+
+  extraInstallCommands = ''
+    mv $out/bin/quiet-${version} $out/bin/${pname}
+  '';
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
Why ask for this change?

`lib` is used three times in `meta`, I don’t see a reason to not use `with lib` here.

Reply:
Although I don't feel this change necessary, it has been done to move this Pull Request forward in: https://github.com/NixOS/nixpkgs/pull/254839/commits/54b9192bc1baf399bd0bf63e2a51a34fb1ad6a59